Lake/Build/Actions.lean

```diff
@@ -43,16 +43,22 @@ def compileOlean (leanFile oleanFile : FilePath)
 def compileOleanAndC (leanFile oleanFile cFile : FilePath)
 (oleanPath : SearchPath := []) (rootDir : FilePath := ".")
 (leanArgs : Array String := #[]) (lean : FilePath := "lean")
+(precompiledPath : SearchPath := ∅)
 : BuildM PUnit := do
   createParentDirs cFile
   createParentDirs oleanFile
+  let mut env := #[("LEAN_PATH", some oleanPath.toString)]
+  if System.Platform.isWindows then
```

Comment:
```suggestion
  if !precompiledPath.isEmpty && System.Platform.isWindows then
```

To avoid `PATH` being set even when no precompilation is being done (as it clogs up the logs quite a bit).